src/parquet/encodings/plain-encoding-test.cc

```diff
@@ -141,7 +144,7 @@ void EncodeDecode<ByteArray, Type::BYTE_ARRAY>::generate_data() {
   int max_byte_array_len = 12 + sizeof(uint32_t);
   size_t nbytes = num_values_ * max_byte_array_len;
   data_buffer_.resize(nbytes);
-  random_byte_array(num_values_, 0.5, data_buffer_.data(), draws_,
+  random_byte_array(num_values_, 0, data_buffer_.data(), draws_,
```

Comment:
Thanks! for changing this. This was throwing warnings in the travis ci build for clang. Should we enable `-Werror`  ?